python/docs/source/getting_started/install.rst

```diff
@@ -130,15 +130,15 @@ to install Spark, for example, as below:
 
 .. code-block:: bash
 
-    tar xzvf spark-3.0.0-bin-hadoop2.7.tgz
+    tar xzvf spark-3.3.0-bin-hadoop3.tgz
 
 Ensure the ``SPARK_HOME`` environment variable points to the directory where the tar file has been extracted.
 Update ``PYTHONPATH`` environment variable such that it can find the PySpark and Py4J under ``SPARK_HOME/python/lib``.
 One example of doing this is shown below:
 
 .. code-block:: bash
 
-    cd spark-3.0.0-bin-hadoop2.7
+    cd spark-3.3.0-bin-hadoop3
```

Comment:
@HyukjinKwon  Here change to use `spark-3.3.0-bin-hadoop3`, WDYT?

Reply:
yeah, I we should change it.